### DISCUSSION TITLE
DOMException: signal is aborted without reason

### DISCUSSION THREAD

**Discussion by PDXmjb** [Unleash/unleash#7100] [Q&A] [answered]
Hi all,

I'm running into an error in multiple apps while using Unleash's React (`@unleash/proxy-client-react v4.2.2`)
 and Next.js (`@unleash/nextjs v1.4.2`) SDKs. In both cases, I receive the following error message client-side:

`Unleash: unable to fetch feature toggles DOMException: signal is aborted without reason`.

I only receive this message when developing locally, and furthermore I do not receive this on _ALL_ calls to `/api/frontend`, but only a subset.

It's more of an annoyance than a real problem, but has anyone else run into this and figured out a solution?

**Comment by ivarconr** [comment, marked as answer]
fixed in https://github.com/Unleash/unleash-proxy-client-js/releases
